.jshintrc

```diff
@@ -20,5 +20,7 @@
 		"Backbone": false,
 		"jQuery": false,
 		"wp": false
-	}
+	},
+
+    "predef": [ "wpsc_vars" ]
```

Comment:
What would happen the the variable instantiate if the WordPress localization inserts the CDATA onto a page, and then we declare the java script global in the code?  Or we declare it multiple times because we have multiple java scripts that use the variable?  Is the behavior consistent across all browser implementations?